test/e2e/common/node/container_probe.go

```diff
@@ -192,7 +192,7 @@ var _ = SIGDescribe("Probing container", func() {
 			FailureThreshold:    1,
 		}
 		pod := livenessPodSpec(f.Namespace.Name, nil, livenessProbe)
-		RunLivenessTest(f, pod, 5, time.Minute*5)
+		RunLivenessTest(f, pod, 3, time.Minute*10)
```

Comment:
only backoffs for 5 retries combined is almost 2 minutes. Plus, the default timeout for a single restart is 4 minutes. So clearly the timeout is not adequate here, should be at least 6 minutes. Since the first restart must be the longest as it involves the pod scheduling, I'd expect 14 minutes can be enough delay for 5 retries (assuming half of time is needed to restart the pod than schedule and start).

```suggestion
                 // ~2 minutes backoff timeouts + 4 minutes defaultObservationTimeout + 2 minutes for each pod restart
		RunLivenessTest(f, pod, 5, 2 * time.Minute + defaultObservationTimeout + 4 * 2 * time.Minute)
```

I think 14 minutes is ok timeout for conformance test.

If we switching to 3 retries, following the same logic we need 30 sec + 4 minutes + 2 * 2 minutes = 8.5 minutes, no need for 10 minutes. I'd prefer 5 retries though just to be on a safe side and not change the conformance test.

Reply:
Done